Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -686,7 +686,7 @@ FReply UUINavWidget::HandleOnKeyUp(FReply Reply, UUINavWidget* Widget, UUINavCom
 		return Reply;
 	}
 
-	if (IsValid(Component) && Component->bIgnoreDueToRebind || Widget->UINavPC->IsListeningToInputRebind())
+	if ((IsValid(Component) && Component->bIgnoreDueToRebind) || Widget->UINavPC->IsListeningToInputRebind())
```

Comment:
The parenthesis should enclose the conditions between the `||`, not between the `&&`